app-backend/app/src/main/scala/toolrun/package.scala

```diff
@@ -0,0 +1,7 @@
+package com.azavea.rf
+
+import com.azavea.rf.datamodel.{PaginatedResponse, ToolRun}
+
+package object toolrun extends RfJsonProtocols {
+    implicit val paginatedToolRunFormat = jsonFormat6(PaginatedResponse[ToolRun])
```

Comment:
Ambiguity might not come up in all cases where there are multiple sufficient implicits within a given scope. There are at least two such cases:
1. [implicit discovery priority](http://stackoverflow.com/questions/1886953/is-there-a-way-to-control-which-implicit-conversion-will-be-the-default-used)
2. (which is relevant here) given `implicit val l: List[T]`; `implicit val x: SomeType`; and `implicit val lx: List[SomeType]` scala should select the most specific (the latter of the three).